src/pages/components/content-group-horizontal.mdx

```diff
@@ -11,15 +11,26 @@ import ComponentFeedback from 'components/ComponentFeedback';
 
 <AnchorLinks>
 
+<AnchorLink>Content guidance</AnchorLink>
 <AnchorLink>Design and functional specifications</AnchorLink>
 <AnchorLink>Development documentation</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)|
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)|
```

Comment:
What is this line referring to?

Reply:
Do you mean the "TOC headline override"? If so, that field came from the need to change just the table of contents headline. Some users thought the headline could be too long for the table of contents so they wanted a field added where they could override the default headline auto-population. @photodow